src/gui/creds/oauth.cpp

```diff
@@ -137,10 +137,10 @@ void OAuth::start()
                     }
                     if (!_expectedUser.isNull() && user != _expectedUser) {
                         // Connected with the wrong user
-                        QString message = tr("<h1>Wrong user</h1>"
-                                             "<p>You logged-in with user <em>%1</em>, but must login with user <em>%2</em>.<br>"
+                        QString message = tr("<h1>Wrong account</h1>"
+                                             "<p>You logged-in with account <em>%1</em>, but must login with account <em>%2</em>.<br>"
```

Comment:
"You logged in with the account... but must log in with the account..." 
- no hyphen in "logged in"
- account --> the account (in both cases)
- login --> log in (2 separate words)

Reply:
On second read this string is a mess, thanks for your suggestions